packages/components/src/card/card/hook.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { cx } from 'emotion';
+
+/**
+ * WordPress dependencies
+ */
+import deprecated from '@wordpress/deprecated';
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useContextSystem } from '../../ui/context';
+import { useSurface } from '../../surface';
+import * as styles from '../styles';
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../types').Props, 'div'>} props
+ */
+function useDeprecatedProps( { elevation, isElevated, ...otherProps } ) {
+	let computedElevation = elevation;
+
+	if ( isElevated ) {
+		deprecated( 'Card isElevated prop', {
+			since: '5.9',
+			alternative: 'elevation',
+		} );
+		computedElevation ??= 2;
+	}
+
+	return {
+		...otherProps,
+		isElevated,
+		elevation: computedElevation,
+	};
+}
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../types').Props, 'div'>} props
+ */
+export function useCard( props ) {
+	const {
+		className,
+		elevation = 0,
```

Comment:
Compared to the g2's "original" implementation, I changed the default value of the `elevation` prop to `0`. This is to match the default value of `false` that the `isElevated` prop has in the old (production) card implementation